packages/e2e/.gitignore

```diff
@@ -0,0 +1,15 @@
+bin
+node.sock
+db
+logs
+*.log
+sockets
+alonzo-private-network/config/network/testnet/cardano-node/config.json
```

Comment:
I think we should just be calling this `local-network`, as it will change as we move to new eras

Reply:
fixed in https://github.com/input-output-hk/cardano-js-sdk/pull/305/commits/d963d0afe05db27b88d1451925c08e851f749877